### PR TITLE
Add gradle wrapper to files for build and update .travis.yml

### PR DESCRIPTION
**Pull request by rhyshort**
- Add gradle wrapper files for use with the build
- Update .travis.yml file to use gradlew instead of gradle

### CODE REVIEW

**Comment by mikerhodes** [comment]
Do the contributing docs need updating to get everyone to use the wrapper? (and how to make sure IDEA/Studio is?)

**Comment by rhyshort** [comment]
The import works exactly the same as far as I can tell, the command line examples should probably be updated to use gradlew

**Comment by mikerhodes** [comment]
Yes, they need to be. Using `gradlew` seems to be the accepted way to do this.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Cloudant Sync - Android is written in Java and uses
 The main requirements are:
 
 * Java 1.6
-* Gradle 1.8+
+* Gradle 2.2.1+
```
Comment (mikerhodes) [inline CONTRIBUTING.md]: Is this still a requirement for most developers if they are instructed to use `gradlew`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -48,6 +48,8 @@ $ brew install gradle
 
 Other platforms TDB.
 
+You can also use the gradle wrapper to download the needed gradle distribution at build time.
+
```
Comment (mikerhodes) [inline CONTRIBUTING.md]: I suggest we remove the gradle section if we're moving to `gradlew`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -65,7 +67,7 @@ Follow the instructions provided on the android developer site.
 The project should build out of the box with:
 
 ```bash
-$ gradle build
+$ ./gradlew build
```
Comment (mikerhodes) [inline CONTRIBUTING.md]: The first time we use gradle, we should mention how to run it on Windows.

**Comment by mikerhodes** [comment]
+1, after squashing.

**Pull request merged**